src/content/documentation/develop/request-the-right-permissions.md

```diff
@@ -96,12 +96,15 @@ Not all [permissions](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExte
 | `sessions`                                                                                                                     | Access recently closed tabs                                                                                                                                                                                      |
 | `tabs`                                                                                                                         | Access browser tabs                                                                                                                                                                                              |
+| `tabHide`                                                                                                                      | Hide and show browser tabs                                                                                                                                                                                           |
 | `topSites`                                                                                                                     | Access browsing history                                                                                                                                                                                          |
-| `webNavigation`                                                                                                                | Access browser activity during navigation                                                                                                                                                                        |
-| Clipboard access:                                                                                                                |
+| `trialML`                                                                                                                      | Download and run AI models on your device                                                                                                                                                                            |
+| `webNavigation`                                                                                                                | Access browser activity during navigation                                                                                                                                                                            |
+| `userScripts`                                                                                                                  | Unverified scripts can pose security and privacy risks. Only run scripts from extensions or sources you trust.                                                                                                       |
```

Comment:
The text you're putting here was the long extra warning that is displayed next to the actual permission message. The permission message should be:

```suggestion
| `userScripts`                                                                                                                  | Allow unverified third-party scripts to access your data                                                                                                 |
```